packages/node_modules/@webex/plugin-meetings/src/meeting/util.js

```diff
@@ -675,12 +675,20 @@ MeetingUtil.setPeerConnectionEvents = (meeting) => {
         Metrics.postEvent({event: eventType.ICE_END, meeting});
         break;
       case ICE_STATE.CONNECTED:
+        meeting.reconnectionManager.iceReconnected();
         LoggerProxy.logger.log('Meeting:util#setPeerConnectionEvents --> ICE STATE CONNECTED.');
         break;
       case ICE_STATE.CLOSED:
         LoggerProxy.logger.log('Meeting:util#setPeerConnectionEvents --> ICE STATE CLOSED.');
         break;
       case ICE_STATE.DISCONNECTED:
+        meeting.reconnectionManager.iceDisconnected()
+          .catch(() => {
+            LoggerProxy.logger.log('Meeting:util#setPeerConnectionEvents --> ICE STATE FAILED RECONNECT. ATTEMPTING MANUAL RECONNECTION.');
```

Comment:
Let's not use `FAILED` here since there is an actual ice state of FAILED. 

```suggestion
            LoggerProxy.logger.log('Meeting:util#setPeerConnectionEvents --> ICE STATE DISCONNECTED. Automatic Reconnection Timed Out.');
```

Reply:
Going to make this change in the next push.